internal/hcs/process.go

```diff
@@ -417,6 +414,16 @@ func (process *Process) Close() (err error) {
 		return nil
 	}
 
+	if process.stdin != nil {
```

Comment:
Same here. Should this happen after hcsCloseProcess?

Reply:
No, this is less compelling -- if Close fails, we still want to clean up as many resources as possible.

Reply:
That's a reasonable expectation as well.